.github/workflows/cargo-test.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         shell: bash
         run: |-
           cd "${{ matrix.dir }}"
-          cargo llvm-cov nextest --workspace --lcov --output-path lcov.info --test-threads=1 --no-fail-fast -P ci 2>&1 | tee /tmp/github-actions.log
```

Comment:
Also taking the chance to do this as per https://nexte.st/docs/features/retries/ since this test too has been flaky this week